package.json

```diff
@@ -17,56 +17,52 @@
   "homepage": "https://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
-    "test:node-require": "node test/require/node.js",
-    "build:browserify-test": "browserify test/require/browserify.js -o build/temp/browserify.js",
-    "build:webpack-test": "webpack --hide-modules test/require/webpack.js build/temp/webpack.js",
-    "copy-fonts": "shx cp -R node_modules/videojs-font/fonts build/temp/font",
-    "copy-dist": "shx cp -R build/temp/* dist/",
-    "copy-examples": "shx cp -R docs/examples dist/",
+    "start": "npm-run-all -p watch karma-server",
+    "clean": "shx rm -rf ./dist ./test/dist ./docs/api",
+    "postclean": "shx mkdir -p ./dist/lang ./test/dist",
+    "changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
+    "build": "npm-run-all build-dev minify zip",
+    "build-dev": "npm-run-all clean lint-errors build:js build:css build:lang build:copy build:test",
```

Comment:
`build:copy` only needs to run on publish, there's nothing in there that's used for dev

Reply:
maybe it's fine to leave it to match current functionality